sdcm/ycsb_thread.py

```diff
@@ -196,11 +197,23 @@ def parse_final_output(result):
         output = {k: str(v) for k, v in output.items()}
         return output
 
+    def db_node_to_query(self, loader):
```

Comment:
did you said you'll have something similar for the regular cassandra-stress ? or it's not needed ? since cql driver is smart  enough to select the closer one on it's own ?

Reply:
I created a task to implement region aware loader for C-S